### ISSUE TITLE
Do you have any plans to develop a payment screen?

### ISSUE THREAD

**Issue by nijin39** [abuanwar072/E-commerce-Complete-Flutter-UI#65]
As the title says, do you have any plans to develop a payment screen? I'm asking because I just happened to update this app to the latest version and it would be nice to have a payment screen.

**Comment by antoniotavareskozan** [comment]
Yes I will build the payment screen and checkout. Next week. Send me your email or WhatsApp to get in touch

**Comment by nijin39** [comment]
Thanks for your reply.

I will develop for backend using AWS.
If I finish, I will share the source.

2023년 6월 21일 (수) 오전 8:02, Antonio Tavares Kozan ***@***.***>님이
작성:

**Comment by AIdevol** [comment]
Will you give me more details about it? I will update by using a Razorpay payment gate for these requirements.
